package.json

```diff
@@ -1,54 +1,54 @@
 {
-  "name": "peacock-ui",
-  "version": "0.0.0-development",
-  "description": "Stencil Component Starter",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.js",
-  "es2015": "dist/esm/index.mjs",
-  "es2017": "dist/esm/index.mjs",
-  "types": "dist/types/components.d.ts",
-  "collection": "dist/collection/collection-manifest.json",
-  "collection:main": "dist/collection/index.js",
-  "unpkg": "dist/peacock-ui/peacock-ui.esm.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/kelvininc/peacock-ui.git"
-  },
-  "files": [
-    "dist/",
-    "loader/"
-  ],
-  "scripts": {
-    "build": "stencil build --docs",
-    "start": "stencil build --dev --watch --serve",
-    "test": "stencil test --spec --e2e",
-    "test.watch": "stencil test --spec --e2e --watchAll",
-    "generate": "stencil generate",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
-    "dev:stencil": "stencil build --watch",
-    "dev": "run-p dev:stencil storybook",
```

Comment:
Probably you can remove the `npm-run-all` dev-dependency. please check it

Reply:
Yes, we most certainly can, will remove it on the upcoming commit.